### ISSUE TITLE
The report is not accurate across timezones

### ISSUE THREAD

**Issue by candrei** [vitaminsoftware/correlator#1]
### Why

Start and end time of Toggl entries are not timezone aware. The report won't be accurate since the `time.mktime` assumes local time while Togl exports time entries in user's timezone. Reports are only accurate when ran within the same timezone as the analyzed user.

### What
- [ ] Use Toggl's [API](https://github.com/toggl/toggl_api_docs) instead of CSV export to retrieve toggled time

**Comment by aismail** [comment]
@candrei yeah that's why you're not in `config.py`. For the rest of the team it seems to be working fine.
